pkg/querier/tripperware/queryrange/split_by_interval_test.go

```diff
@@ -377,6 +377,10 @@ func Test_evaluateAtModifier(t *testing.T) {
 				[2m:])
 			[10m:])`,
 		},
+		{
+			in:       `irate(kube_pod_info{namespace="test"}[1h:1m] @ start())`,
+			expected: `irate(kube_pod_info{namespace="test"}[1h:1m] @ 1546300.800)`,
+		},
```

Comment:
Can we add one more test where we have both @ modifer on the vector selector and subquery?